lib/swagger_yard/operation.rb

```diff
@@ -170,6 +173,19 @@ def sort_parameters
       @parameters.sort_by! {|p| p.name}
     end
 
+    ##
+    # Example:
+    # @extension x-internal: true
+    def add_extension(tag)
+      key, value = tag.text.split(":").map(&:strip)
```

Comment:
Small thing, I'd suggest using the "limit" argument too, like `split(":", 2)`, so we don't spit it in more parts and end up with truncated `value`.
```
> "x-example-url: http://www.example.com".split(":")
=> ["x-example-url", " http", "//www.example.com"]
> "x-example-url: http://www.example.com".split(":", 2)
=> ["x-example-url", " http://www.example.com"]
```


```suggestion
      key, value = tag.text.split(":", 2).map(&:strip)
```